packages/pluggableWidgets/datagrid-date-filter-web/src/DatagridDateFilter.editorConfig.ts

```diff
@@ -19,6 +20,9 @@ export function getProperties(values: DatagridDateFilterPreviewProps, defaultPro
     if (!values.adjustable) {
         hidePropertyIn(defaultProperties, values, "screenReaderButtonCaption");
     }
+    if (!values.advanced) {
+        hidePropertiesIn(defaultProperties, values, ["onChange", "valueAttribute"]);
```

Comment:
What's the diff between `hidePropertyIn` & `hidePropertiesIn` ? besides doing multiple at the same time.?